packages/affiliates/libs/affiliates.js

```diff
@@ -66,14 +68,19 @@ const DeployerRewards = ({ queries, empCreatorAbi, empAbi, coingecko, synthPrice
   }
 
   // returns array of tuples [emp address, deployer address]
-  async function getEmpDeployerHistory(address, start, end) {
-    // this query is relatively small
-    const logs = await queries.getLogsByContract(address, start, end);
+  async function getEmpDeployerHistory(address) {
+    // this query is relatively small but expensive, gets all logs from begginning of time
+    const logs = await queries.getAllLogsByContract(address);
     const decode = DecodeLog(empCreatorAbi);
-    return logs.map(decode).reduce((result, log) => {
-      result.push([log.args.expiringMultiPartyAddress, log.args.deployerAddress]);
-      return result;
-    }, []);
+    return logs
+      .map(log => decode(log, log))
+      .reduce((result, log) => {
+        result.push([
+          log.args.expiringMultiPartyAddress,
+          { deployer: log.args.deployerAddress, timestamp: log.block_timestamp, number: log.block_number }
```

Comment:
this just adds additional meta data as we processs the deployer logs. Could be used to optimize balance queries by getting deployment time for each emp contract. Currently though its not used.